packages/elements-next/src/index.mdx

```diff
@@ -7,10 +7,105 @@ route: /
+- CSS should look the same as the Bulma equivient where the component is equivient. Eg an input or a button in v1 should look identical in v2. We are actively hiring a designer and the styles will change in time but we want this to be simple to do when the time comes.
+- The docs should be styled the same as Elements - Elements docs should look like Elements!
+- There should be far fewer components. We have several different ways of doing notifications - there should be a single approach. See below for list of components to migrate.
+- HTML output should be next to the mounted React Component and the Rendered Component in the docs in all cases.
+- All variants of components should be visible at the top of the page in the generated props section. They should all be editable at the bottom, although only the basic component without variants should be rendered by default.
+- Where we export multiple components from the same file, we should export them as modules eg `<Headings.Subheading>`
+- Should work in Edge (latest version before Chromium), Edge Chromium, Firefox, Chrome, Safari
+- Should use Linaria `cx` for composing classes.
+
+### Components to migrate
+
+The following components to be built in v2. Will add tickets to each one as they are written.
+
+- Alert, Info and Helper to be combined into a single module with variant components
+- Button as a module with all variants including button group
+- Input as a module to include; Text, Number, Password, Number, Select, Image Upload (there are two of these, should be combined), Checkbox, Radio, DatePicker, MultiSelect (vanilla refactor), HTML Editor, File Input, Text Area
+- Dynamic links, tidied and refactored if required.
+- HTML Renderer
+- Helper guide - can be massively tidied and refactored
+- Headings - already have a start on this, to tidy up
+- Lists - to include UL, OL and Icon Lists
+- Section with super flexible variants to act as containers for any element
+- Container, for pages and top level elements - as per Section, super flexible with modifiers
+- Grid, Grid Items - should be based on native CSS Grid, basis 12 cols
+- Loader - inline and page centered variants
+- Maps - consider ditching React Wrappers for Google maps - it's trivial to load a map in a UseEffect hook and saves tonnes of JS
+- Menu - shoud be refactored to better support actions, icons and sub menus as a dropdown
+- Notification, Toast, ToastMessage, all combined - single module to notify users, simple implementation with variants. Current notification is the most flexible.
+- Modal, single, simple implementation
+- Pagination
+- Spreadsheet - tidy and refactor as required
+- Table - just a simple styled HTML table, we don't use any advanced functionality and if we need it, just use Spreadsheet
+- Form - to export Formik, to include form Headings and Helpers
```

Comment:
I think we don't need to include Formik to the elements V2, just need to export form components like Input, Select, CheckBox,... It's better if users can choose the form framework they want, Formik is not the best for every case.

Reply:
@nphivu414 So I left Formik in because I thought that was the team's preference. I agree with you, I think it would be nicer if we were Form Lib agnostic. Happy to take this out, would be interested if anyone else has a view though?